nglview/show.py

```diff
@@ -281,6 +286,29 @@ def show_schrodinger_structure(mol, **kwargs):
     return NGLWidget(structure_trajectory, **kwargs)
 
 
+def show_biopython(mol, **kwargs):
+    '''Show NGL widget with Biopython structural entity.
+
+    Takes a Structure, Model, Chain, Residue or Atom
+    from Bio.PDB as its data input.
+
+    Examples
```

Comment:
update `biopython` in [README](https://github.com/arose/nglview/blob/master/README.md)